src/storage/gpu_swap_prefetch.h

```diff
@@ -0,0 +1,44 @@
+#ifndef GPU_SWAP_PREFETCH_H
+#define GPU_SWAP_PREFETCH_H
+
+#include <mxnet/gpu_swap_history.h>
+
+#if MXNET_USE_CUDA
+#include <cuda_runtime.h>
+#endif
+
+namespace mxnet {
+
+class Prefetch {
+public:
+
+  ~Prefetch();
+  static Prefetch* Get();
+  static std::shared_ptr<Prefetch> _GetSharedRef();
+  void StartPrefetching();
+  void StopPrefetching();
+  void HistoryBasedPrefetch(int device);
```

Comment:
Again, put this function declaration to line 22 and add a comment before the line: Prefetch algorithm declarations.